specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.4-preview.1/common.json

```diff
@@ -69,6 +69,59 @@
       },
       "description": "The key vault server error.",
       "x-nullable": true
+    },
+    "LifetimeActions": {
+      "properties": {
+        "trigger": {
+          "$ref": "#/definitions/LifetimeActionsTrigger",
+          "description": "The condition that will execute the action."
+        },
+        "action": {
+          "$ref": "#/definitions/LifetimeActionsType",
+          "description": "The action that will be executed."
+        }
```

Comment:
Which of these are required? I assume one or both, which should be added to a `required` array property in this model.

Reply:
Just FYI. It is same as we had for keys, now we are adding it to secrets.

Reply:
Never mind. I misremembered we punted key rotation to 7.4 but see it is [destined for 7.3](https://github.com/Azure/azure-rest-api-specs/blob/main/specification/keyvault/data-plane/Microsoft.KeyVault/preview/7.3-preview/keys.json). Note, however, that it's in servers' customers' best interest to be clear on which properties are required for future models.